ProjectStructureAndRoles.md

```diff
@@ -6,29 +6,36 @@
+The purpose of an API Repository is the definition and documentation of one or multiple (very closely related) APIs which will be always released together. Each API repository must have a README.MD file (with a description of the API scope and a link to the Sub Project home page), a CODEOWNERS and a MAINTAINERS.MD file. An API repository should have at least one Maintainer. Ideally a Sub Project is managed by two or more Maintainers, depending on the size and scope of the API. Here, the responsibilities must be clearly agreed upon between all of the Maintainers. This includes coordinating who is responsible for which issues and pull requests. 
+The Contributors to an API repository are recorded by the mechanisms of GitHub in the Sub Project’s repositories. Each Contributor, Codeowner and Maintainer should also be listed in the [PARTICIPANTS.MD](./PARTICIPANTS.MD) file in the Governance repository root directory. Each API repository should have a license file, a GOVERNANCE.MD file (pointing to the Governance repository) and subdirectories /documentation/API_documentation, /documentation/SupportingDocuments and /code/API_definitions. Only Codeowners should have write permissions to an API repository.
 
-Some Sub Projects may have distinct (although sometimes overlapping) sets of Contributors, Codeowners and Maintainers.
+A Sub Project should have at least one Maintainer. Ideally a Sub Project is managed by two or more Maintainers, depending on the size and scope of the Sub Project. The Maintainers of a Sub Project are defined by the union of the Maintainers listed within the MAINTAINERS.md files of the API Repositories which are belonging to the Sub Project. It is recommended that there is an overlap between the Maintainers of the API repositories of the Sub Project, but it is not mandated that all Maintainers of the Sub Project take responsibility for all APIs in the Sub Project.
+  
+A Provider Implementation repository only must have a CODEOWNERS file, a license file, a GOVERNANCE.MD file (pointing to the Governance repository) and a subdirectory /code/API_code with API code of the transformation function Northbound --> Southbound. Only Codeowners should have write permissions to a Provider Implementation (PI) repository. Provider Implementation repositories have the same name as the API repository but with suffix _-PI§x§_.
 
-As document standard for documents in the main branch of Sub Projects only Markdown format shall be used. Only exceptions are supporting documents, for these other formats like PDF or PPT are allowed (Note: Documents which are meant as base for discussion or to explain change requests are best attached to an issue).
+Some Sub Projects may have distinct (although sometimes overlapping) sets of Contributors, Codeowners, and Maintainers.
```

Comment:
```suggestion
Some Sub Project repositories may have distinct (although sometimes overlapping) sets of Contributors, Codeowners, and Maintainers.
```

Reply:
The sentence was actually unchanged and addressed the relation between Sub Projects. For the relation between the repository *within* a Sub Project the overlap is even recommended within the content in the paragraphs above.

As this sentence here does not add any additional guidance I propose to delete it from the document.

Reply:
OK. Fine to delete it. I thought it was trying to say that the different repositories that form a Sub Project do not need to have common contributors and maintainers, though indeed that should be encouraged.

Reply:
Deletion of the sentence done.